core/storage/file/gae_models_test.py

```diff
@@ -0,0 +1,197 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for core.storage.file.gae_models."""
+
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(file_models,) = models.Registry.import_models([models.NAMES.file])
+
+
+class FileMetadataModelTest(test_utils.GenericTestBase):
+    """Tests the FileMetadataModel class."""
+
+    def test_get_new_id(self):
```

Comment:
Maybe a better name is test_get_new_id_raises_not_implemented_error. In general, try to make the test names descriptive, so that the purpose of the test is evident from the name itself.

Reply:
Done.